develop/api_metadata.rst

```diff
@@ -366,3 +333,231 @@ Caveats:
+
+Tables on all Nodes
+-------------------
+
+Citus has other informational tables and views which are accessible on all nodes, not just the coordinator.
+
+.. _pg_dist_authinfo:
+
+Connection Credentials Table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+  This table is a part of Citus Enterprise Edition. Please `contact us <https://www.citusdata.com/about/contact_us>`_ to obtain this functionality.
+
+The ``pg_dist_authinfo`` table holds authentication parameters used by Citus nodes to connect to one another.
+
++----------+---------+-------------------------------------------------+
+| Name     | Type    | Description                                     |
++==========+=========+=================================================+
+| nodeid   | integer | Node id from :ref:`pg_dist_node`, or 0, or -1   |
++----------+---------+-------------------------------------------------+
+| rolename | name    | Postgres role                                   |
++----------+---------+-------------------------------------------------+
+| authinfo | text    | Space-separated libpq connection parameters     |
++----------+---------+-------------------------------------------------+
+
+Upon beginning a connection, a node consults the table to see whether a row with the destination ``nodeid`` and desired ``rolename`` exists. If so, the node includes the corresponding ``authinfo`` string in its libpq connection. A common example is to store a password, like ``'password=abc123'``, but you can review the `full list <https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-PARAMKEYWORDS>`_ of possibilities.
+
+The parameters in ``authinfo`` are space-separated, in the form ``key=val``. To write an empty value, or a value containing spaces, surround it with single quotes, e.g., ``keyword='a value'``. Single quotes and backslashes within the value must be escaped with a backslash, i.e., ``\'`` and ``\\``.
+
+The ``nodeid`` column can also take the special values 0 and -1, which mean *all nodes* or *loopback connections*, respectively. If, for a given node, both specific and all-node rules exist, the specific rule has precedence.
+
+::
+
+    SELECT * FROM pg_dist_authinfo;
+
+     nodeid | rolename | authinfo
+    --------+----------+-----------------
+        123 | jdoe     | password=abc123
+    (1 row)
+
+Connection Pooling Credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+  This table is a part of Citus Enterprise Edition. Please `contact us <https://www.citusdata.com/about/contact_us>`_ to obtain this functionality.
+
+The ``pg_dist_poolinfo`` table holds the host, port and database name for Citus to use when connecting to a node through a connection pooler. This table is specific to connection pooling rather than direct connections between nodes, and overrides the values in :ref:`pg_dist_node <pg_dist_node>`.
```

Comment:
What do you think of the following reframing: "If you want to use a connection pooler to connect to a node, you can set it up using this metadata table. If pool information is present, Citus will try to use that instead of the direct connection to the information in pg_dist_node. In certain scenarios (e.g. rebalancing) CItus will connect directly since logical replication..."

Reply:
I like it. Worked it in.